risc0/zkvm/src/exec/mod.rs

```diff
@@ -183,7 +184,9 @@ impl<'a> Executor<'a> {
                         syscalls,
                         exit_code,
                         log2_ceil(total_cycles.next_power_of_two()),
+                        idx,
```

Comment:
```suggestion
                        segments.len(),
```

Reply:
Fixed in: c041649754ba2478d005ad8b77cd01fe1c8102d5 with additional removal of the idx var.